scripts/make-analysis-and-display.sh

```diff
@@ -65,7 +67,7 @@ tippecanoe \
     --minimum-zoom 0 --maximum-zoom 16 \
     --drop-smallest-as-needed \
     --force --output "$WORKDIR/conflated.mbtiles" \
-    "$WORKDIR/conflated.geojson"
+    "$WORKDIR/orma-sections.geojson"
```

Comment:
@batpad we may want to use the same filename because i'm not sure if the tileset layer name will change.